be/src/storage/inverted/clucene/clucene_inverted_writer.cpp

```diff
@@ -45,7 +45,7 @@ class CLuceneInvertedWriterImpl : public CLuceneInvertedWriter {
 
     explicit CLuceneInvertedWriterImpl(const std::string& field_name, const std::string& directory,
                                        const TabletIndex* inverted_index)
-            : _directory(std::move(directory)), _inverted_index(inverted_index) {
+            : _directory(std::move(directory)), _inverted_index(inverted_index), _wstring_buffer(1024 * 1024, '\0') {
```

Comment:
`_wstring_buffer(1024 * 1024, '\0')` this will introduce a memset of 2MB memory for every _wstring_buffer.